test/utils.js

```diff
@@ -1,12 +1,13 @@
+const BN = require('bn.js');
 const crypto = require("crypto");
 const { EthereumConnection, toChecksummedAddress } = require("@iov/ethereum");
 
-async function getBalanceApproximation(address) {
-  const connection = await EthereumConnection.establish("http://localhost:7545");
+async function getBalance(address) {
+  const connection = await EthereumConnection.establish("http://localhost:8545");
   const account = await connection.getAccount({ address });
   const ethWallet = account && account.balance.find(({ tokenTicker }) => tokenTicker === "ETH");
   const balance = ethWallet ? ethWallet.quantity : "0";
-  return parseInt(balance, 10);
+  return new BN(balance, 10);
```

Comment:
We should call `connection.disconnect()` here before returning, just to prepare for upcoming websocket implementation.